code/game/objects/items/devices/deskbell.dm

```diff
@@ -77,7 +77,10 @@
 	return
 
 /obj/item/device/deskbell/attack_hand(var/mob/user)
-	ring()
+	if(anchored)
+		ring()
+	else
+		..()
```

Comment:
Unless you want people ringing deskbells in their backpack, which really isn't intended, and having to click-drag the deskbell to themselves to pick it up which is unintuitive and silly, (actually the cause of the issue is assuming you can't pick it up) this is necessary.